examples/rxJavaExample/README.md

```diff
@@ -0,0 +1,16 @@
+# Examples combining RxJava and Realm
+
+# Things to keep in mind
+
+- Observables might have a default `Scheduler` they operate on that is different than the one 
+  the RealmObject was created on. Accessing Realm objects on the wrong thread will throw a 
+  `IllegalStateException`.
+  
+- RealmObjects are live objects that automatically stay up to date. Observables that operate 
```

Comment:
Since you put /version/ within citation marks, it might be a good idea to explain it more carefully.